web/cobrands/buckinghamshire/assets.js

```diff
@@ -242,9 +252,16 @@ fixmystreet.assets.add($.extend(true, {}, defaults, {
                 enable_report_form();
             } else {
                 // User has clicked a road that Bucks don't maintain.
+
+                var map = {
+                    "HE": '#js-not-council-road-he',
+                    "HWOA": '#js-not-council-road-other'
```

Comment:
Is this missing `P`/`HWSA`? Clicking at, e.g. `/report/new?longitude=-0.8269637409057568&latitude=51.819437574604656` doesn't result in the right message being shown:
![image](https://user-images.githubusercontent.com/4776/49812751-02c4b880-fd5e-11e8-9f5a-8c0b59e95a2f.png)